packages/core/src/textInput/styles/LabelInner.ts

```diff
@@ -8,7 +8,7 @@ const LabelInner = styled.label(({ theme: { color, transitionDuration } }) => ({
   lineHeight: rem(12),
   color: color.trunks[100],
   top: rem(11),
-  zIndex: 1,
+  zIndex: 3,
```

Comment:
Do we need this z-index. What happens when you click on this label, will it jump to top?